.github/workflows/android.yml

```diff
@@ -360,19 +364,113 @@ jobs:
             echo "Pulling unit test results..."
             adb pull /data/local/tmp/ov_test/unit_test_results.xml ./unit_test_results.xml || { echo "Warning: Could not pull unit test results"; adb shell "ls -la /data/local/tmp/ov_test/"; adb shell "cat /data/local/tmp/ov_test/unit_test_results.xml" || echo "No unit test results file"; }
 
-      - name: Upload test results
+      - name: Upload unit test results
         uses: actions/upload-artifact@330a01c490aca151604b8cf639adc76d48f6c5d4 # v5.0.0
         if: always()
         with:
-          name: cpu_tests_results_android_x64
+          name: cpu_unit_tests_results_android_x64
+          path: ./unit_test_results.xml
+          if-no-files-found: 'ignore'
+
+  CXX_Func_Tests:
+    needs: [Smart_CI, Docker, Build]
+    timeout-minutes: 120
+    runs-on: aks-linux-4-cores-16gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_test.ubuntu_22_04_android }}
+      volumes:
+        - /mount:/mount
+      options: --device /dev/kvm
+    env:
+      ANDROID_API_LEVEL: 30
+      ANDROID_TARGET: google_apis
+      ANDROID_ARCH: x86_64
+      INSTALL_DIR: android_package
+      INSTALL_TEST_DIR: android_tests
+      GTEST_FILTER: '*EltwiseLayer*'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+
+      - name: Download CPU test binaries
+        uses: actions/download-artifact@018cc2cf5baa6db3ef3c5f8a56943fffe632ef53 # v6.0.0
+        with:
+          name: ov_cpu_tests_android_x64
+          path: ${{ env.INSTALL_TEST_DIR }}
+
+      - name: Download OpenVINO runtime package (Android)
+        uses: actions/download-artifact@018cc2cf5baa6db3ef3c5f8a56943fffe632ef53 # v6.0.0
+        with:
+          name: ov_android_package_x64
+          path: ${{ env.INSTALL_DIR }}
+
+      - name: Setup KVM permissions
+        run: |
+          /usr/local/bin/setup-kvm.sh || true
+          ls -la /dev/kvm || echo "KVM device not available"
```

Comment:
This step only logs /dev/kvm and keeps going. If KVM/nested‑VT is missing, the emulator silently falls back to TCG and tests stop running natively on the host CPU. Please hard‑fail when KVM is absent (e.g., `test -c /dev/kvm || exit 1` or `emulator -accel-check | grep -q KVM || exit 1`) to guarantee “command‑for‑command” execution.